Fixing dist file

joshwnj opened the pull request:
- Removes dist file from git (as suggested in #18)
- Adds `npm run build`, which is also run on `prepublish`

joshwnj commented:
@kof would you change anything here?

kof commented:
I would add installation instructions for bower users like I did it here: https://github.com/jsstyles/jss/blob/master/docs/setup.md

joshwnj commented:
Great suggestion, thanks @kof 

I've also:

- updated the build script so browserify produces a standalone umd script
- added `example-umd` to show how to use it with plain `<script>` tags
